source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -4,14 +4,34 @@ source:
   file: steps-open-mongodb-view.yaml
   ref: open-mongodb-view
 ---
-title: Open the |vsce| *Overview* page from the Command Palette or from 
-       the :guilabel:`MongoDB` view in the :guilabel:`Activity Bar`.
+title: Open the |vsce| Overview page
 level: 4
 ref: create-connection
 content: |
 
+  You can use the the MongoDB view or the Command Palette to open the 
+  overview page. Use the tabs below to see each procedure:
```

Comment:
[Minor]

> Use the tabs below to see each procedure:

Generally we don't want to use "below" to locate information within a topic.

https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#b

Maybe we can remove this sentence entirely, since it should be intuitive that users should select the appropriate tab.

Reply:
Ah thanks for the reference, yes I agree this can be removed entirely.